src/Writing/OpenAPISpecWriter.php

```diff
@@ -241,6 +233,9 @@ protected function generateEndpointRequestBodySpec(OutputEndpointData $endpoint)
                 $schema['properties'][$name] = $fieldData;
             }
 
+            if (array_key_exists('properties', $schema)) {
```

Comment:
Let's put that in the code:

```suggestion
            // We remove 'properties' if the request body is an array, so we need to check if it's still there
            if (array_key_exists('properties', $schema)) {
```